trinity/plugins/builtin/ethstats/ethstats_client.py

```diff
@@ -29,16 +36,15 @@ def __init__(
         self,
         websocket: websockets.client.WebSocketClientProtocol,
         node_id: str,
-        *args,
-        **kwargs,
+        token: CancelToken = None,
     ) -> None:
-        super().__init__(*args, **kwargs)
+        super().__init__(token)
 
         self.websocket = websocket
         self.node_id = node_id
 
-        self.send_queue: asyncio.Queue = asyncio.Queue()
-        self.recv_queue: asyncio.Queue = asyncio.Queue()
+        self.send_queue: asyncio.Queue[EthstatsMessage] = asyncio.Queue()
+        self.recv_queue: asyncio.Queue[EthstatsMessage] = asyncio.Queue()
```

Comment:
Did you try running the plugin after that change? Not sure if this works with Python 3.7 but at least with Python 3.6 you need to put that in quotes `asyncio.Queue[EthstatsMessage]'` which is an annoying limitation that applies to some generic builtin types.

Reply:
Working correctly for me in Python 3.6.6, but will add quotes around `EthstatsMessage` just in case.